packages/dd-trace/src/llmobs/tagger.js

```diff
@@ -313,11 +313,15 @@ class LLMObsTagger {
 
   // any public-facing LLMObs APIs using this tagger should not soft fail
   // auto-instrumentation should soft fail
-  _handleFailure (msg) {
+  _handleFailure (msg, errorTag) {
     if (this.softFail) {
       log.warn(msg)
     } else {
-      throw new Error(msg)
+      const err = new Error(msg)
+      if (!errorTag) {
```

Comment:
```suggestion
      if (errorTag) {
```
i might be reading it wrong but i think we want to add it if it does exist? lmk if i'm not getting it 😅